src/js/components/projects/table/index.tsx

```diff
@@ -26,9 +28,18 @@ const ProjectTable = ({
     ...project,
     numCollaborators: project.github_users?.length || 0,
   }));
+  const statusOrder = {
+    [PROJECT_STATUSES.REVIEW]: 1,
+    [PROJECT_STATUSES.IN_PROGRESS]: 2,
+    [PROJECT_STATUSES.PLANNED]: 3,
+    [PROJECT_STATUSES.MERGED]: 4,
+  };
+  const projectDefaultSort = sortBy(items, [
+    (item) => statusOrder[item.status],
+  ]);
```

Comment:
@wlonk I did this sort on the front-end, but then realized that (unlike tasks) projects are paginated... so for this to work, I think we actually want to apply this to the default model `ordering` on the back end -- so it happens pre-pagination. Is that something we can do? Ideally, sort by `status` (review, then in-progress, then planned, then merged), then by `-created_at`, finally by `name`.

Reply:
That is absolutely something we can do. Do we want it to be a default ordering applied in all cases unless specifically overridden? If so, we just add it to the `Meta` on that model.

Reply:
Commit to relevance-order in.